src/server/pbsd_init.c

```diff
@@ -733,10 +737,11 @@ pbsd_init(int type)
 				printf("%s\n", log_buffer);
 				return -1;
 			}
-			svr_save_db(&server, SVR_SAVE_NEW);
-		} else {
-			svr_save_db(&server, SVR_SAVE_NEW);
 		}
+		svr_save_db(&server, SVR_SAVE_NEW);
+		dflt_scheduler = sched_alloc(PBS_DFLT_SCHED_NAME);
+		(void)sched_save_db(dflt_scheduler, SVR_SAVE_NEW);
```

Comment:
I think I gave this comment last time when a pull was raised related to multi scheduler - Shouldn't we set the default and then save it?

Reply:
@arungrover 
I think we also had discussed about the same earlier. Please find the following link for more details.
https://github.com/PBSPro/pbspro/pull/324#discussion_r125700738